pkg/sdk/tags_validations.go

```diff
@@ -39,20 +38,20 @@ func (opts *createTagOptions) validate() error {
 }
 
 func (v *AllowedValues) validate() error {
-	if ok := validateIntInRange(len(v.Values), 1, 50); !ok {
-		return fmt.Errorf("number of the AllowedValues must be between 1 and 50")
+	if !validateIntInRange(len(v.Values), 1, 50) {
+		return errIntBetween("AllowedValues", "Values", 1, 50)
 	}
 	return nil
 }
 
 func (v *TagSet) validate() error {
 	var errs []error
 	if !exactlyOneValueSet(v.MaskingPolicies, v.Comment) {
-		errs = append(errs, errOneOf("MaskingPolicies", "Comment"))
+		errs = append(errs, errOneOf("TagSet", "MaskingPolicies", "Comment"))
 	}
 	if valueSet(v.MaskingPolicies) {
-		if ok := validateIntGreaterThanOrEqual(len(v.MaskingPolicies.MaskingPolicies), 1); !ok {
-			errs = append(errs, fmt.Errorf("number of the MaskingPolicies must be greater than zero"))
+		if !validateIntGreaterThanOrEqual(len(v.MaskingPolicies.MaskingPolicies), 1) {
```

Comment:
why not validateIntGreaterThan? (it would be more consistent with the error which is only greater, and not greater or equal)

Reply:
Yeah, I didn't see that I have to confirm then which one is correct

Reply:
True, I just automatically refactored it as is. Changed.